pkg/oc/cli/admin/policy/modify_roles.go

```diff
@@ -577,6 +579,17 @@ func (o *RoleModificationOptions) AddRole() error {
 	}
 	existingSubjects := roleBinding.Subjects()
 	newSubjects := addSubjects(o.Users, o.Groups, o.Subjects, existingSubjects)
+	systemGroups := map[string]bool{
```

Comment:
There are a lot more well known virtual groups.  I am wondering if this is worth maintaining.   That being said without an exact list we will miss typos.  Certainly the easiest thing to check for is a `:` colon in the group name but I feel like that will make the overall check less useful.

Reply:
I'm unsure what you suggest is the best course of action.

Also, if we don't know all the virtual groups so we can keep this updated, that sounds like a problem in and of itself.